net/instaweb/rewriter/strip_subresource_hints_filter.cc

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright 2015 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Author: kspoelstra@google.com (Kees Spoelstra)
```

Comment:
This should be your email address at we-amp or wherever; kspoelstra@google.com isn't you, is it?

(If you're not comfortable putting your email in the code then it should just be "Author: Kees Spoelstra".)